Server/src/Config/Config.java

```diff
@@ -65,4 +67,66 @@ public String getProperty(String property) {
     return config.getProperty(property);
   }
 
+  private Properties loadDefaultConfigFile() {
+    Properties defaultProperties = null;
+
+    try {
+      InputStream defaultConfig = Config.class.getResourceAsStream(configFileName);
+      if (defaultConfig == null) {
+        throw new FileNotFoundException("Couldn't find default configuration at " + configFileName);
+      }
+      defaultProperties = new Properties();
+      defaultProperties.load(defaultConfig);
+
+      defaultConfig.close();
+    } catch (FileNotFoundException ex) {
+      ex.printStackTrace();
+    } catch (IOException ex) {
+      ex.printStackTrace();
+    }
+
+    return defaultProperties;
+  }
+
+  public boolean shouldUpdateUserConfig() {
+    boolean shouldUpdateUserConfig = false;
+    try {
+      if (config == null) {
+        throw new Exception("User's config file is null");
+      }
+
+      Properties defaultConfig = loadDefaultConfigFile();
+      Set<String> defaultVariables = defaultConfig.stringPropertyNames();
+      Set<String> userVariables = config.stringPropertyNames();
+      if (!defaultVariables.equals(userVariables)) {
+        shouldUpdateUserConfig = true;
+      }
+    } catch (Exception ex) {
+      ex.printStackTrace();
+    }
+
+    return shouldUpdateUserConfig;
+  }
+
+  public void updateUserConfig() {
+
+    Properties defaultConfig = loadDefaultConfigFile();
+    Properties newValuesToAppend = new Properties();
+    defaultConfig.forEach((key, value) -> {
+      if (!config.containsKey(key.toString())) {
+        newValuesToAppend.put(key.toString(), value.toString());
+        System.out.println("added " + key.toString());
```

Comment:
This should either be removed or include more info about the context of what's happening, eg "User config missing X, adding default value".